setup.py

```diff
@@ -17,7 +17,15 @@
 
 if os.environ.get('USE_NCCL', '1') == '1':
     cxx_flags.append('-DFMOE_USE_NCCL')
-    ext_libs.append('nccl')
+    if os.environ.get('USE_ROCM', '0') == '1':
+        ext_libs.append('rccl')
```

Comment:
Is this all we need to use RCCL instead of NCCL? Is there any code (especially API names) that needs to be changed (and thus guarded by something like `FMOE_USE_RCCL`?

Reply:
The api of rccl lib is same as nccl's.  The name is different, rccl is librccl.so and nccl is libnccl.so

Reply:
Got it, thanks!